code/game/turfs/open.dm

```diff
@@ -109,13 +109,14 @@
 	air.temperature += temp
 	air_update_turf()
 
-/turf/open/freon_gas_act()
+/turf/open/freon_gas_act(real_freon)
 	for(var/obj/I in contents)
 		if(!HAS_SECONDARY_FLAG(I, FROZEN)) //let it go
 			I.make_frozen_visual()
-	for(var/mob/living/L in contents)
-		if(L.bodytemperature <= 50)
-			L.apply_status_effect(/datum/status_effect/freon)
+	if(real_freon)
```

Comment:
please apply this to all cold if you're going to nerf muh freon to just be "plasma but starts cold"